docs/manual-CN/deploy-cluster.md

```diff
@@ -0,0 +1,146 @@
+---
+
+本节介绍`Nebula`的部署
+---
+
+#### 下载并安装包
+
+目前，Nebula官方提供 `CentOS 7.5`， `CentOS 6.5`，`Ubuntu 1604`和`Ubuntu 1804`包，rpm或deb包下载点击[此处](https://github.com/vesoft-inc/nebula/releases)。
+
+ `CentOS`系统：
+
+```
+rpm -ivh nebula-{VERSION}.{SYSTEM_VERSION}.x86_64.rpm
+```
+
+ `Ubuntu`系统：
+
+```
+dpkg -i nebula-{VERSION}.{SYSTEM_VERSION}.amd64.deb
+```
+
+配置文件默认目录为`/usr/local/nebula/etc`，请更改`meta_server_addrs`，配置 Meta Server 的地址。
+
+启动多副本 Meta 服务需将多个地址配置到`meta_server_addrs`，地址间需使用逗号分隔。
+
+使用`data_path`设置`Meta`的底层存储路径。
+***
+
+#### 启动 Nebula 集群
+
+
+目前，nebula 集群由`scripts/services.sh`运维，可使用此脚本`start`，`stop`或 `restart`重启集群。
+
+示例命令如下：
+
+```
+scripts/services.sh <start|stop|restart|status|kill>
+```
+
+metas， storages 和 graphs 包含其自身的 hosts。
+
+***
+
+#### 配置引用
+
+**Meta Service** 支持如下配置属性：
+
+属性名               | 默认值            | 说明
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon 监听端口
+`reuse_port`                | true                     | 开启 SO_REUSEPORT 选项
+`data_path`                 | ""                       | 根数据目录
+`meta_server_addrs`                     | ""                       | 一系列由逗号分隔的IP地址，IP数与副本数相等，如果为空，则表明副本数为1
+`local_ip`                  | ""                       | 指定本地IP  NetworkUtils::getLocalIP.
+`num_io_threads`            | 16                       | IO 线程数
+`meta_http_thread_num`      | 3                        | meta daemon 的 http 线程数
+`num_worker_threads`        | 32                       | worker 数
+`part_man_type`             | memory                   | memory，meta
+`pid_file`                  | "pids/nebula-metad.pid"  | 存储进程 ID 的文件
+`daemonize`                 | true                     | 作为 daemon 进程运行
+`cluster_id`                | 0                        | 每个集群都惟一的 ID
```

Comment:
Do we have this configuration?

Reply:
> Do we have this configuration?

who can confirm this? @dangleptr @critical27